### DISCUSSION TITLE
Make / Model / Color

### DISCUSSION THREAD

**Discussion by olympia** [algertc/ALPR-Database#23] [Ideas]
It's not absolutely essential, but having this information added to the known plates database, along with the ability to filter and search it, could come quite handy at times.

**Comment by algertc** [comment]
This is limited by the AI capability right now. See this [thread](https://ipcamtalk.com/threads/full-alpr-database-system-for-blue-iris.79082/post-892688) for extra info.

The column is still in the DB. If the AI can detect it better, it will be super quick to add.

**Comment by olympia** [comment]
Sure, but I meant to add the ability to add it manually to the known plates.

**Reply by algertc** [reply]
Oh, yes. I can definitely do this. Of course it's a lot more useful when its automated, but I understand not wanting to put that stuff in the notes column when there's a dedicated column for it.

I removed it from the frontend in the update since it wasn't doing anything, but I'll add a selector to show/hide columns in the next one that will let users configure what they would like to see.  I'll add a vehicle description field to the known plates dialog box.

**Comment by algertc** [comment]
This is still on my radar and MikeLud has commented on its practicality on ipcamtalk. We will get there eventually.
